### DISCUSSION TITLE
MPV cannot read filepaths from commands

### DISCUSSION THREAD

**Discussion by Christian-Nerd** [mpv-player/mpv#15492] [Q&A] [answered]
I want to automatically play video and audio from a course that I have installed on my system. Because each video/audio file is in their own separate folders, I use this command 
```
mpv "$(find ~/Documents/[Path to Course]/[Path to first unit -iname "*mp3" | awk -F "\n" 'BEGIN{ORS=" "} {gsub(" ", "\\ "); print $1}')"
```
to separate the file paths. This did not work even when I added single and double quotes as MPV read all these files wrong. When it used double/single quotes it read multiple files as one. When I escaped all spaces, then it read each word as a filepath. How do I get MPV to read filepaths from commands?

# System Information
![image](https://github.com/user-attachments/assets/7cd256ab-33c5-4f1f-8407-b51372a244ee)

**Comment by guidocella** [comment]
`find | mpv --playlist=-`

**Reply by Christian-Nerd** [reply, marked as answer]
The solution was using find -print0 and xargs -0 and it worked.
